src/User/pages/Privacy-Policy/privacy.css

```
@@ -8,42 +8,43 @@ body {
 }
 
 .containerprivacy {
-    max-width: 80%;
-    margin: 90px auto;
-    padding: 20px;
-    /* background: #f6ebe4; */
-    border-radius: 8px;
-    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    max-width: 85%;
+    padding: 15px 30px 30px 30px;  /* 👈 same spacing jaise terms */
+    padding: 30px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)